packages/server-wallet/src/models/channel.ts

```diff
@@ -173,6 +177,67 @@ export class Channel extends Model implements RequiredColumns {
     });
   }
 
+  public static directFundingStatus(
```

Comment:
I think this deserves a unit test with decent coverage.

Reply:
Unit tests added https://github.com/statechannels/statechannels/pull/2950/files#diff-7d310f70f103b57a4ab31608a210eda379528970c7804f919e58389b440584d6